debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlSnapshotChangeEventSource.java

```diff
@@ -407,8 +407,10 @@ protected String enhanceOverriddenSelect(RelationalSnapshotContext<MySqlOffsetCo
     @Override
     protected String getSnapshotSelectColumns(TableId tableId) {
         Table table = databaseSchema.tableFor(tableId);
```

Comment:
This line can be moved into `getPreparedColumnNames` in the super class and then there is no need to pass in a `Table` but only the `TableId`.  This effectively should make this method and all the other methods for the other connector implementations one-liners.

Reply:
Reverted back to this implementation as Postgres doesn't support `HistorizedRelationalDatabaseSchema` and `RelationalSnapshotChangeEventSource` accepts `HistorizedRelationalDatabaseSchema`.

Reply:
@Naros Logged https://issues.redhat.com/browse/DBZ-3818 for `RelationalSnapshotChangeEventSource` to support `RelationalDatabaseSchema`.